src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -149,6 +149,21 @@ function rrule(::typeof(*), A::AbstractMatrix{<:Real}, B::AbstractMatrix{<:Real}
     return A * B, times_pullback
 end
 
+function rrule(::typeof(*), A::Real, B::AbstractArray{<:Real})
+    function times_pullback(Ȳ)
+        return (NO_FIELDS, @thunk(dot(Ȳ, B)), @thunk(A' * Ȳ))
```

Comment:
Why `A'` rather than just `A`?

Reply:
Yeah fair, with this restricted to `Real` it is the same.

```suggestion
        return (NO_FIELDS, @thunk(dot(Ȳ, B)), @thunk(A * Ȳ))
```